obspy/io/cybershake/tests/test_core_cybershake.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
```

Comment:
usually we just call this file `test_core.py` it being clear it's about cybershake by the folder structure. just cosmetics but might as well stay in line.